app/controllers/api/v1/podcasts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::PodcastsController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def index
+    render json: Podcast.all, adapter: :json
+  end
+
+  def show
+    render json: Podcast.where(id: params[:id]), adapter: :json
+  end
+
+  def create
+    data = JSON.parse(request.body.read)
+    new_podcast = Podcast.create(name: data["name"], publisher: data["publisher"], description: data["description"], link: data["link"])
+    render json: new_podcast
```

Comment:
alas, we should be using strong params here, and we shouldnt be doing `request.body.read`. If you are having trouble accessing params, then we should dig into that further

Error handling would be nice too, but we could table that if need be

Reply:
Wait.....are you even using this endpoint? If not remove it, because I think you are doing this in a regular controller